mozci/push.py

```diff
@@ -411,32 +487,14 @@ def get_candidate_regressions(self, runnable_type):
                     passing_runnables.add(name)
                     continue
 
-                fixed_by_commit_classification_notes = [
-                    n[:12]
-                    for c, n in summary.classifications
-                    if c == "fixed by commit"
-                    if n is not None
-                ]
-                if len(fixed_by_commit_classification_notes) > 0:
-                    if (
-                        self.backedout
-                        and self.backedoutby[:12]
-                        in fixed_by_commit_classification_notes
-                    ):
-                        # If the failure was classified as fixed by commit, and the fixing commit
-                        # is a backout of the current push, it is definitely a regression of the
-                        # current push.
-                        candidate_regressions[name] = (-math.inf, summary.status)
-                        continue
-                    elif all(
-                        HGMO.create(n, branch=self.branch).is_backout
-                        for n in fixed_by_commit_classification_notes
-                    ):
-                        # If the failure was classified as fixed by commit, and the fixing commit
-                        # is a backout of another push, it is definitely not a regression of the
-                        # current push.
-                        passing_runnables.add(name)
-                        continue
+                classification = self._check_classification(
```

Comment:
nit: let's call this variable `at_fault` or something similar.

Reply:
I've renamed both the variable and the function in 32dbc9d.